website/docs/r/config_remediation_configuration.html.markdown

```diff
@@ -0,0 +1,138 @@
+---
+layout: "aws"
+page_title: "AWS: aws_config_remediation_configuration"
+sidebar_current: "docs-aws-resource-config-remediation-configuration"
+description: |-
+  Provides an AWS Config Remediation Configuration.
+---
+
+# Resource: aws_config_remediation_configuration
+
+Provides an AWS Config Remediation Configuration.
+
+~> **Note:** Config Remediation Configuration requires an existing [Config Rule](/docs/providers/aws/r/config_config_rule.html) to be present. Use of `depends_on` is recommended (as shown below) to avoid race conditions.
```

Comment:
If we set the `config_rule_name` of the remediation to the `name `value` of the rule, then we can get the built-in Terraform dependency management without using `depends_on`